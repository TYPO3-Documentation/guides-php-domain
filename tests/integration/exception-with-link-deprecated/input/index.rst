=============
PHP Exception
=============

..  php:namespace:: TYPO3\CMS\Core\Exception\Page

..  php:exception:: BrokenRootLineException

    Exception for root line traversal when a page within the root line traversal
    is missing / can not be resolved.

See also :php:exc:`TYPO3\CMS\Core\Exception\Page\BrokenRootLineException`.
