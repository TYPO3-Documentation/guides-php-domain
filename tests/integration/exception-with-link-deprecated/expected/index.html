<!-- content start -->
    <div class="section" id="php-exception">
            <h1>PHP Exception</h1>
            <dl class="php exception">
    <dt class="sig sig-object php"
        id="typo3-cms-core-exception-page-brokenrootlineexception">
        <em class="property"><span class="pre">exception</span> </em>
            <span class="sig-prename descclassname"><span class="pre">\TYPO3\CMS\Core\Exception\Page\</span></span>
<span class="sig-name descname"><span class="pre">BrokenRootLineException</span></span><a class="headerlink" href="#typo3-cms-core-exception-page-brokenrootlineexception" title="Permalink to this definition">Â¶</a>    </dt>
    <dd>

    <p>Exception for root line traversal when a page within the root line traversal
is missing / can not be resolved.</p>

    </dd>
</dl>

    <p>See also <a href="/index.html#typo3-cms-core-exception-page-brokenrootlineexception">\TYPO3\CMS\Core\Exception\Page\BrokenRootLineException</a>.</p>

    </div>
<!-- content end -->
